docs/05-dev-tools/wallets/metamask.mdx

```diff
@@ -0,0 +1,175 @@
+---
+sidebar_label: MetaMask
+sidebar_position: 3
+title: Configure MetaMask Wallet for Rootstock
+tags: [metamask, rootstock, tools, rsk, wallets, guides]
+description: "Learn how to connect to Rootstock with a MetaMask Wallet"
+---
+
+import WrappedAddNetworkButtons from "/src/components/AddNetworkButtons";
+
+In this guide, you will learn how to set up Rootstock on Metamask.
+
+## Option 1: Add Rootstock Network to MetaMask Automatically
+
+1. Install MetaMask on your browser from the oficial website: [https://metamask.io/download/](https://metamask.io/download/)
+2. Click on the buttons below to add Rootstock Mainnet and Rootstock Testnet
+
+<WrappedAddNetworkButtons />
+
+<div class="container">
+  <div class="row">
+    <div class="col col--6">
+      <div class="col-demo">
+        <button id="add-network-button-mainnet" class="btn">
+          <span class>
+            <div class="flex items-center text-10 px-1">
+              <img class="mr-1" width="35" src="/img/metamask/metamask.svg"></img>
+              Add Rootstock Mainnet to MetaMask
+            </div>
+          </span>
+        </button>
+      </div>
+    </div>
+    <div class="col col--6">
+      <div class="col-demo">
+        <button id="add-network-button-testnet" class="btn">
+          <span class>
+            <div class="flex items-center text-10 px-1">
+              <img class="mr-1" width="35" src="/img/metamask/metamask.svg"></img>
+              Add Rootstock Testnet to MetaMask
+            </div>
+          </span>
+        </button>
+      </div>
+    </div>
+  </div>
+</div>
+
+## Option 2: Add Rootstock Network to MetaMask Manually
+
+1. Install Metamask on your browser from the oficial website: [https://metamask.io/download/](https://metamask.io/download/)
+2. Open the MetaMask extension.
+3. In the network selector (top left corner), choose Add Network.
+
+{" "}
+
+<div styles="text-align: center">
+  <img class="metamask-screenshot" src="/img/metamask/metamask-1.png"></img>
+</div>
+
+{" "}
+
+<div styles="text-align: center">
+  <img class="metamask-screenshot" src="/img/metamask/metamask-2.png"></img>
+</div>
+
+4. Click on Add Network Manually
+5. Fill with the requiered fields to add Rootstock Mainnet or Testnet
+
+{" "}
+
+<div styles="text-align: center">
+  <img class="metamask-screenshot" src="/img/metamask/metamask-3.png"></img>
+</div>
+
+<table class="table">
```

Comment:
This network information table can come before the image. Use a header 2 to specify that its network settings

Also please populate the image with the testnet values and screenshot.

Thanks

Reply:
ok, changes applied. Didn't use header 2 because it will take it out of the 5 steps of the manual configuration... unless you prefer the table to be on another section, below Option 2.